docs/proposals/algorithms/lifelong-learning/Cloud-Robotic AI Benchmarking for Edge-cloud Collaborative Lifelong Learning.md

```diff
@@ -0,0 +1,297 @@
+
+paper link: [The Cityscapes Dataset for Semantic Urban Scene Understanding](https://www.cityscapes-dataset.com/wordpress/wp-content/papercite-data/pdf/cordts2016cityscapes.pdf)
+
+Below shows one example figure(RGB) in the dataset.
+
+![aachen_000000_000019_leftImg8bit](images/aachen_000000_000019_leftImg8bit.png)
+
+#### 2.1.2 SYNTHIA dataset
+
+The ***SYNTHetic** collection of Imagery and Annotations* is a dataset that has been generated with the purpose of aiding semantic segmentation and related scene understanding problems in the context of driving scenarios. SYNTHIA consists of a collection of photo-realistic frames rendered from a virtual city and comes with precise pixel-level semantic annotations for 13 classes: misc, sky, building, road, sidewalk, fence, vegetation, pole, car, sign, pedestrian, cyclist, lane-marking.
+
+paper link: [The SYNTHIA Dataset: A Large Collection of Synthetic Images for Semantic Segmentation of Urban Scenes](www.cv-foundation.org/openaccess/content_cvpr_2016/papers/Ros_The_SYNTHIA_Dataset_CVPR_2016_paper.pdf)
+
+Below shows one example figure(RGB) in the dataset.
+
+![1769](images/1769.png)
+
+### 2.2 Introduction for the benchmarking  website
+
+#### 2.2.1 Content of the benchmarking website
+
+| Page               | Content                                |
+| ------------------ | -------------------------------------- |
+| Home page          | 1.Dataset overview                     |
+|                    | 2.Lifelong learning algorithm overview |
+|                    | 3.Data sample display                  |
+| Documentation page | 1.Dataset partition description        |
+|                    | 2.Data statistics                      |
+|                    | 3.Data format                          |
+|                    | 4.Data annotation                      |
+| Download page      | 1.Instructions and links               |
+| Benchmark page     | 1.Various algorithm and metric results |
+
+#### 2.2.2 An Example
+
+![web](images/web.png)
+
+### 2.3 New test examples of semantic segmentation for lifelong learning
+
+#### 2.3.1 Why we need it?
+
+The way the dataset is divided during each round of training in lifelong learning tasks has a significant impact on the final model training results. If only a random partitioning method is used, there may be situations that are inconsistent with reality, such as later data appearing in earlier training sets. At the same time, changes in data distribution in different splitting methods can make it difficult to compare test results. Therefore, we urgently need a reasonable dataset splitting method as a benchmark for testing lifelong learning algorithms on semantic segmentation datasets. 
+
+Currently, we have implemented the "City Mode" test benchmark, and we will implement more various test benchmarks in the future.
+
+#### 2.3.2 An example of "City Mode" test benchmark
```

Comment:
As mentioned on the SIG AI routine meeting, the travel mode has been modified as city mode, which takes different vehicles of different cities as mobile nodes. The current version looks good to me.